metrics/run-stat-reports.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Does this script need to be in the plane codebase?

Reply:
hmm I use it in the dockerfile

Reply:
I think I could add the nats username password stuff into the runargs, then this script can go. But I think that can go in a follow up pr.